Providers/Profile.swift

```diff
@@ -924,7 +924,8 @@ open class BrowserProfile: Profile {
                     guard let self = self else { return }
                     let devices = state.remoteDevices.map { d -> RemoteDevice in
                         let t = "\(d.deviceType)"
-                        return RemoteDevice(id: d.id, name: d.displayName, type: t, isCurrentDevice: d.isCurrentDevice, lastAccessTime: d.lastAccessTime, availableCommands: nil)
+                        let lastAccessTime = d.lastAccessTime == nil ? nil : UInt64(clamping: d.lastAccessTime!)
```

Comment:
Stable Kotlin doesn't have unsigned types, so the autogenerated API bindings can't have unsigned types, so we need to do this weird type conversion here. Sorry :-(